cypress/integration/profile/update-tls.spec.ts

```diff
@@ -0,0 +1,178 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2021
+ * SPDX-License-Identifier: Apache-2.0
+ **********************************************************************/
+// Tests the update of a profile
+import { profileFixtures } from '../../fixtures/profile'
+import { apiResponses, httpCodes } from '../../fixtures/api/apiResponses'
+import { wirelessFixtures } from 'cypress/fixtures/wireless'
+// ---------------------------- Test section ----------------------------
+
+describe('Test Profile Page', () => {
+  // Real stack execution order: profile/create-tls.spec.ts, wireless/create.spec.ts and then profile/update-tls.spec.ts
+  beforeEach('clear cache and login', () => {
+    cy.setup()
+    cy.myIntercept('GET', 'profiles?$top=25&$skip=0&$count=true', {
+      statusCode: httpCodes.SUCCESS,
+      body: apiResponses.profiles.getAll.success.response
+    }).as('get-profiles')
+
+    cy.goToPage('Profiles')
+    // Stub the get and post requests
+    cy.myIntercept('GET', 'ciraconfigs?$count=true', {
+      statusCode: httpCodes.SUCCESS,
+      body: apiResponses.ciraConfigs.getAll.forProfile.response
+    }).as('get-configs')
+
+    cy.myIntercept('GET', 'wirelessconfigs?$count=true', {
+      statusCode: httpCodes.SUCCESS,
+      body: apiResponses.wirelessConfigs.getAll.forProfile.response
+    }).as('get-wirelessConfigs')
+
+    cy.myIntercept('GET', 'profiles?$top=25&$skip=0&$count=true', {
+      statusCode: httpCodes.SUCCESS,
+      body: apiResponses.profiles.getAll.empty.response
+    }).as('get-profiles')
+
+    cy.goToPage('Profiles')
+    cy.wait('@get-profiles')
+
+    // change api response
+    cy.myIntercept('GET', 'profiles?$top=25&$skip=0&$count=true', {
+      statusCode: httpCodes.SUCCESS,
+      body: apiResponses.profiles.getAll.success.response
+    }).as('get-profiles2')
+    cy.myIntercept('GET', 'happyTlspath', {
+      statusCode: httpCodes.SUCCESS,
+      body: apiResponses.profiles.getAll.success.response.data
+    }).as('get-profile')
+  })
+
+  it('Update profile to Admin Control Mode', () => {
+    cy.myIntercept('PATCH', 'profiles', {
+      statusCode: httpCodes.SUCCESS,
+      body: profileFixtures.happyPathTls
+    }).as('patch-profile')
+
+    cy.get('mat-row').contains(profileFixtures.happyPathTls.profileName).click()
+    cy.wait('@get-profile')
+    cy.get('mat-select[formcontrolname=activation').click()
+    cy.contains('Admin Control Mode').click({ force: true })
+    cy.setAMTMEBXPasswords(Cypress.env('AMTPASSWORD'), Cypress.env('MEBXPASSWORD'))
+    cy.contains('SAVE').click({ force: true })
```

Comment:
why are these  forced?

Reply:
I was seeing flaky test runs when I was developing. I can try to remove it now and see if they work.